src/textual/widgets/_markdown.py

```diff
@@ -644,6 +645,18 @@ async def load(self, path: Path) -> bool:
         self.update(markdown)
         return True
 
+    def unhandled_token(self, token: Token) -> MarkdownBlock | None:
+        """Process an unhandled token.
+
+        Args:
+            token: The token to handle.
+
+        Returns:
+            Either a widget to be added to the output, or `None`.
+        """
+        del token
```

Comment:
Why the `del` here? Pretty sure it won't do anything since there will be a reference in the caller.

Reply:
Just silence linters about the unused parameter; fairly normal idiom for that. It keeps the name for the documentation but tells linters and the like that it's not mistakenly unused.

Reply:
Haven't see that one. I'm not keen on code which does something pointless to keep a linter happy. Is there another way to quell the error?

Reply:
It's the best one I've seen, similar to using `assert` to give a big hint for typing. The alternatives I'm aware of are just not paying attention to the linter, doing `_ = that_unused_argument`, or including linter-choice-specific pragma comments.

Happy to go with an alternative you're more comfortable with.

Reply:
I think I'd just go for a pragma comment here.

Reply:
Thinking about it, I don't think there's a pragma that covers pylint, pyright and pylance (to pick 3 that will complain about an unused argument). I guess given it's happening a lot in the rest of Textual, it's preferable to just let them complain.